custom/abdm/milestone_one/views/abha_creation_views.py

```diff
@@ -53,6 +53,6 @@ def verify_mobile_otp(request):
     resp = abdm_util.verify_mobile_otp(otp, txn_id)
     if resp and "txnId" in resp:
         resp = abdm_util.create_health_id(txn_id)
-        resp.pop("token")
+        resp["user_token"] = resp.pop("token")
```

Comment:
I could not find the reason on the spec for this, why are we adding the user_token to specific endpoints?

Reply:
This token is used in the request sent for generating the health card.
From the doc,
```
Endpoint: /abdm/api/get_health_card_png
Method: POST
Payload: {"user_token": <user token received in previous step>}

```

Reply:
Cool. Can you add it to the spec doc?
https://docs.google.com/document/d/1m7QGxCLIkQNTs8pbkHEjS_L2KGoUj1tAfVAmlzDSMnY/edit

Can you share why were removing the token in response?
is it available in response for only specific end points to ABDM? Thinking if we should just return it for all endpoints, when available and ABDM app should store it when it gets it.

Reply:
It's already part of the spec and above snippet is taken from there itself (3. New endpoint to get health card in PNG format).

> Can you share why were removing the token in response?

It is a big string and was removed to make the response light since at that point we were not using it anywhere.

> is it available in response for only specific end points to ABDM

It is for auth endpoints, where we confirm the OTPs afaik.

> Thinking if we should just return it for all endpoints, when available and ABDM app should store it when it gets it.

We have covered such known endpoints in this PR.